publish.js

```diff
@@ -473,6 +473,21 @@ var copyRecursiveSync = function(src, dest) {
     }
 };
 
+function getVersions () {
+    var topLevelOutDir = path.normalize(env.opts.destination);
```

Comment:
Good catch. Same setup over here. 

It’s kinda sad cause I would love to automate this. 
But I guess there is no way around defining versions in the jsdoc-configuration file :pensive:

I would prefer something like:
```
"versionSwitcher": {
    "versions": [
        "1.0.0",
        "1.0.1"
    ]
}
```

Cause the URLs are still relative, right?

We could have leave an option like that to auto generate version like it is done right know:
```
"versionSwitcher": {
    "versions": [
        "1.0.0",
        "1.0.1"
    ]
}
```

Or does it make that too complex for a user?